indexing-service/src/test/java/org/apache/druid/indexing/worker/shuffle/LocalIntermediaryDataManagerAutoCleanupTest.java

```diff
@@ -144,8 +146,8 @@ private boolean isCleanedUpAfter2s(String supervisorTaskId, Period timeoutPeriod
     // Start the data manager and the cleanup cycle
     intermediaryDataManager.start();
 
-    // Check the state of the partition after 2s
-    Thread.sleep(2000);
+    // Check the state of the partition after 3s
+    Thread.sleep(3000);
```

Comment:
Thanks for fixing this, @abhishekagarwal87 . I guess the 2s was causing some flakiness if the cleanup tick was missed.